### DISCUSSION TITLE
有没有办法通过什么命令，可以直接激活我创建的模块。

### DISCUSSION THREAD

**Discussion by ribychen** [LSPosed/LSPosed#1958] [Q&A] [answered]
目前激活模块，需要手动在模块列表里面，找到自己的模块，然后点激活。这样才能有效激活模块。
有更方便的方式吗，可以通过命令或者修改那部分代码，可以实现对外一个接口来实现，快速激活模块呢？
我目前尝试root权限修改数据库modules_config.db，但是这样首次无效，之后手动点击激活模块之后，我下次可以修改数据库来激活对应的应用。

**Comment by yujincheng08** [comment, marked as answer]
写个模块来偷偷激活自己hook支付宝偷用户支付密码对吧

**Reply by ribychen** [reply]
不是，我的目的是要自动化搬砖的时候，应用升级，能批量激活所有手机的模块。
不然我的，一台一台手机手动点击。。

**Reply by yujincheng08** [reply]
那不是一样？不可能支持的。

**Reply by ribychen** [reply]
谢谢，那有办法通过命令行固定包名启动LSPosed界面吗？ 我脚本去点击。
目前我没找到lsposed的包名。
